UPGRADE-3.4.md

```diff
@@ -69,6 +69,9 @@ Process
  * The `Symfony\Component\Process\ProcessBuilder` class has been deprecated,
    use the `Symfony\Component\Process\Process` class directly instead.
 
+ * The `Symfony\Component\Process\Process` constructor's $cwd argument will throw 
```

Comment:
`$cwd` should also be enclosed with backticks

Reply:
I think we should actually reword this (please also add the same entry to the `CHANGELOG.md` file of the `Process` component):

> Passing a not existing working directory to the constructor of the `Symfony\Component\Process\Process` class is deprecated and will not be supported anymore in Symfony 4.0.

And please also add a similar entry to the `UPGRADE-4.0.md` file:

> Passing a not existing working directory to the constructor of the `Symfony\Component\Process\Process` class is not supported anymore.

Reply:
Are these messages verbatim?

Reply:
Done